src/app/constants/routes.js

```diff
@@ -0,0 +1,14 @@
+export const ROUTES = [
+  {
+    name: "HOME",
+    path: "/",
+  },
+  {
+    name: "FAVORITE",
+    path: "/favorite",
```

Comment:
Shouldn't these be plural?
```suggestion
    name: "FAVORITES",
    path: "/favorites",
```

Reply:
Done